.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Context
+
+<!-- Why do we need this PR? What was the reason that led you to make this change? -->
```

Comment:
But comments in the code are visible haha. I like @tkyi 's idea of putting it in italics and still having it be visible.